packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -182,13 +159,14 @@ export class OdspDocumentService implements IDocumentService {
                 this.epochTracker,
                 // flushCallback
                 async () => {
-                    if (this.currentConnection !== undefined && !this.currentConnection.disposed) {
-                        return this.currentConnection.flush();
+                    const currentConnection = this.odspDocumentServiceDelayLoaded?.currentDeltaConnection;
+                    if (currentConnection !== undefined && !currentConnection.disposed) {
+                        return currentConnection.flush();
                     }
                     throw new Error("Disconnected while uploading summary (attempt to perform flush())");
                 },
                 () => {
```

Comment:
can you please add a comment here on what is the type of this callback (similar to above - // flushCallback)